submit-web/src/components/SubmissionItem/ConsultationRecord/index.tsx

```diff
@@ -173,15 +184,17 @@ export const ConsultationRecord = () => {
     callSaveSubmission({
       data: {
         type: SUBMISSION_TYPE.FORM,
+        status: status || SUBMISSION_STATUS.COMPLETED.value,
```

Comment:
I don't love that we are defaulting here, we shouldn't default at all, the status should be passed mandatorily